packages/harpstrata/src/harpstrata/HarpStrata/types/index.ts

```diff
@@ -1,6 +1,6 @@
 import type { PozitionIds } from '../../Pozition'
 import type { PitchIds, PitchMatrix } from '../../Pitch'
-import type { ActiveIds, IsActiveComplex } from '../../IsActive'
+import type { ActiveIds, ActiveDegreeIds, ActivePitchIds } from '../../IsActive'
```

Comment:
IsActive is not the best folder name for this anymore. I don't know if it should be changed now or in another feature where we reorganise the logic in general.

Reply:
The renaming will be more obvious in the next feature branch. I'll not worry about it now.